docs/application/native/guides/machine-learning/machine-learning-training.md

```diff
@@ -0,0 +1,469 @@
+# Machine Learning Training
+
+Machine Learning Training API allows you to construct, control, and train a machine learning model in Tizen devices.
+
+The main features of the Machine Learning Training API include:
+
+1. Construction of deep neural network(DNN)
```

Comment:
Please add a space before bracket:

1. Construction of deep neural network (DNN)